runtime/doc/eval.txt

```diff
@@ -2408,6 +2408,15 @@ shellescape({string} [, {special}])
 				String	escape {string} for use as shell
 					command argument
 shiftwidth([{col}])		Number	effective value of 'shiftwidth'
+sign_define({name} [, {dict}])	Number	define or update a sign
+sign_getdefined([{name}])	List	get a list of defined signs
```

Comment:
Currently "name" appears to be unique, i.e. `sign_getdefined` could return a dict with the names as keys.
But on the other hand this could also be changed (later) to make names only unique together with groups?!